scripts/python/offset_viewer.py

```diff
@@ -1,191 +0,0 @@
-"""GUI tool to visualize offset analysis when analyzing system latency."""
```

Comment:
Did these scripts get relocated somewhere, or are they no longer used?

Reply:
The functionality will be moved into the repository itself - in the short run we'll be without it. There are tickets to support offsets and photodiodes in the backlog.